frontend/apps/filemanager/filemanager.lua

```diff
@@ -819,6 +825,7 @@ function FileManager:reinit(path, focused_file)
     if self.select_mode then
         self.title_bar:setRightIcon("check")
     end
+    self:handleFile(focused_file, callback)
```

Comment:
Same expliciteness required : `self:handleFileManagerDocument(focused_file, filemanagerdocument_handle_callback )`
(Given the edgy context, I don't mind this being long :)